crates/bevy_winit/src/lib.rs

```diff
@@ -600,6 +586,10 @@ pub fn winit_runner(mut app: App) {
                     },
                     _ => {}
                 }
+
+                if window.is_changed() {
+                    cache.window = window.clone();
+                }
```

Comment:
This is the key change.
We update the cached window after receiving new info from `winit` to avoid seeing these as changes we need to send back to `winit` again.